relayer-cli/package.json

```diff
@@ -23,6 +24,10 @@
     "web3-batched-send": "^1.0.3"
   },
   "devDependencies": {
-    "ts-node": "^10.9.2"
+    "@types/jest": "^29.5.14",
+    "jest": "^29.7.0",
+    "ts-jest": "^29.2.5",
+    "ts-node": "^10.9.2",
+    "yarn": "^1.22.22"
```

Comment:
Is `yarn` needed to be defined in devDependencies? Also the version differs from the global one, `4.2.2`

Reply:
@mani99brar somehow my local environment decided to add it. It was not intentional. I'm removing it.

Reply:
Removed in the latest commit